tidy3d/components/medium.py

```diff
@@ -1267,6 +1336,257 @@ def elements(self) -> Dict[str, IsotropicMediumType]:
         return dict(xx=self.xx, yy=self.yy, zz=self.zz)
 
 
+class FullyAnisotropicMedium(AbstractMedium):
+    """Fully anisotropic medium.
```

Comment:
maybe add "including all 9 components of the permittivity and conductivity tensors." Also, mention whether it can model dispersive materials?

Reply:
added more description